bucket/openark.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "1.1.0",
+    "description": "An anti-rookit(ARK) tool",
+    "homepage": "https://openark.blackint3.com",
+    "license": "LGPL-2.1-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/BlackINT3/OpenArk/releases/download/v1.1.0/OpenArk64.exe",
+            "hash": "6fad8b0553d0ab5464f2547b8890696b0bcf4e1d8ea89b29e6c9840389b3a95e"
+        },
+        "32bit": {
+            "url": "https://github.com/BlackINT3/OpenArk/releases/download/v1.1.0/OpenArk32.exe",
+            "hash": "550056c99590a902d9df4a2dfbf75da9f045dd8fb25099b30b34f0ca3937a2e1"
+        }
+    },
+    "pre_install": "Rename-Item \"$dir\\OpenArk*.exe\" \"OpenArk.exe\"",
+    "bin": "OpenArk.exe",
```

Comment:
Available command line arguments?

Reply:
No, just we can open UI from command line.